.github/workflows/translations-sync.yml

```diff
@@ -0,0 +1,36 @@
+name: Sync translations
+
+on:
+  workflow_dispatch:
+  push:
+    paths: ['bridge_ui/locales/**']
+    branches: [master]
+
+permissions:
+  contents: write
+  pull-requests: write
```

Comment:
This is required so that Crowdin can open PRs in this repo. We need to make sure that GitHub Actions are allowed to do that. This can be toggled in the settings of this repo: https://github.com/alephium/wormhole-fork/settings/actions

It looks like this:

![image](https://github.com/alephium/wormhole-fork/assets/1579899/3b341d9e-2ceb-4e87-8683-1514a33cc074)